### PR TITLE
Pomelo.EnityFrameworkCore.MySql intergation test

### PR DESCRIPTION
**Pull request by Kielek**
## Why

Increase test coverage

Related to #2331

## What

Add integration tests with Pomelo.EnityFrameworkCore.MySql v 6.0.0 and v.7.0.0.
Technically it is covered by the other testing library, but it looks like pretty popular lib. v7 was downloaded more than 300k in 2 months https://www.nuget.org/packages/Pomelo.EntityFrameworkCore.MySql

## Tests

CI

## Checklist

- ~~[ ] `CHANGELOG.md` is updated.~~
- ~~[ ] Documentation is updated.~~
- [x] ~~New features are covered by tests.~~ Extended test coverage.

### CODE REVIEW

**Review by rajkumar-rangaraj — APPROVED** [review]

File: OpenTelemetry.AutoInstrumentation.sln [hunk 1]
```diff
@@ -168,7 +168,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DependencyListGenerator", "
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GacInstallTool", "tools\GacInstallTool\GacInstallTool.csproj", "{6A63DAA1-463A-4F7E-B3FF-3B444F161DBD}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "LibraryVersionsGenerator", "tools\LibraryVersionsGenerator\LibraryVersionsGenerator.csproj", "{119F5BAD-6A58-40EA-8E0A-666CADE3CAF8}"
```
Comment (pjanotti) [inline OpenTelemetry.AutoInstrumentation.sln:171]: I __think__ I introduced these `FAE0...` via `dotnet sln add` - anyway, how are you detecting these?

File: test/test-applications/integrations/TestApplication.EntityFrameworkCore.Pomelo.MySql/TestContext.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using Microsoft.EntityFrameworkCore;
+
+namespace TestApplication.EntityFrameworkCore.Pomelo.MySql;
+
+public class TestContext : DbContext
```
Comment (pjanotti) [inline test/test-applications/integrations/TestApplication.EntityFrameworkCore.Pomelo.MySql/TestContext.cs:21]:
```suggestion
public class DbTestContext : DbContext
```

**Review by pjanotti — APPROVED** [review]

File: OpenTelemetry.AutoInstrumentation.sln (hunk 1 above)
Comment (Kielek) [reply OpenTelemetry.AutoInstrumentation.sln:171]: It was modified by VisualStudia when I added new project.

File: test/test-applications/integrations/TestApplication.EntityFrameworkCore.Pomelo.MySql/TestContext.cs (hunk 2 above)
Comment (Kielek) [reply test/test-applications/integrations/TestApplication.EntityFrameworkCore.Pomelo.MySql/TestContext.cs:21]: It was copy paste from the second EntityFrameworkCore project.

I can fix both places, but I preffer `TestDbContext`. If you are fine with current state, please just resolve comment.

**Pull request merged**